src/packages/recompose/compose.js

```diff
@@ -1,17 +1,19 @@
 export default function compose(...funcs) {
-  if (funcs.length === 0) {
+  const filteredFuncs = funcs.filter(func => typeof func === 'function')
```

Comment:
this is the only line that really changed in this file. The rest of the changes are just because of the lint rule against re-assignment of parameters. Not disagreeing on the rule, just explaining. I tried modifying the parameter name to make the change less invasive, but 14-16 tests rely on that parameter name being 'funcs'